File: tecnicell.client/src/app/Interfaces/business/ApiResponses/PhoneRepairResponse.ts
import { Image } from "../Models/Image";
import { PhoneRepairHistory } from "../Models/PhoneRepair";

export interface PhoneRepairResponse {
    view:      PhoneRepairView;
    image:     Image;
    histories: PhoneRepairHistory[];
}

export interface PhoneRepairView {
    code:              string;
    type:              string;
    name:              string;
    customerName:      string;
    customerId:        string;
    customerNumber:    string;
    price:             number;
    imageCode:         string;
    date:              Date;
    currentState:      string;
    actionDescription: string;
}

File: Tecnicell.Server/Models/Responses/PhoneRepairResponse.cs
using Tecnicell.Server.Models.Entity;
using Tecnicell.Server.Models.ViewModel.Phone;
using Tecnicell.Server.Models.ViewModel;

namespace Tecnicell.Server.Models.Responses
{
    public class PhoneRepairResponse
    {
        public PhoneRepairView? View { get; set; }
        public ImageViewModel? Image { get; set; }
        public IEnumerable<PhoneRepairHistoryViewModel> Histories { get; set; } = new List<PhoneRepairHistoryViewModel>();
    }
}
